ui/components/KubeStatusIndicator.tsx

```diff
@@ -54,7 +53,7 @@ function KubeStatusIndicator({
       <Icon
         size="base"
         type={icon}
-        text={short ? readyText : computeMessage(conditions)}
+        text={short || suspended ? readyText : computeMessage(conditions)}
```

Comment:
Pipe into ternary can get a little weird to think about and is hard to read. We ignore `suspended` if `short` is truthy?

Can we "write this dumb" using an if statement so it is easier to understand?